### DISCUSSION TITLE
What's the best method/workflow when swapping with stylized art or when there is something on the face?

### DISCUSSION THREAD

**Discussion by bconnor218** [glucauze/sd-webui-faceswaplab#118] [Q&A]
I love this tool, it works wonderfully about 90% of the time!

I have been playing around with other artistic styles and things and I've noticed that often when the face is swapped it doesn't always conform to the artistic style.  Or in other instances is there is like a cut or bruise on the face or some other type of paint stroke or texture to the skin, these features aren't carried over properly to the swapped face.

What is the best method to preserve these things from the original image onto the swapped image?  I'm mostly performing post-processing inpainting, but that doesn't always seem to get me all the way there, especially after Codeformer runs fix the face.  

What are others doing?

**Comment by glucauze** [comment]
The insightface is targeted for realistic images. There is no real way to fix that. You could try inpainting. As for object like glasses on the face, img2img with mask might work sometimes.
